### PR TITLE
Integrate prefetch with PerformanceResourceTiming deliveryType.

### PR DESCRIPTION
**Pull request by jeremyroman**
This builds on w3c/resource-timing#343 to return a delivery type of `"navigational-prefetch"` for navigation timing entries delivered from a matching prefetch record.

### CODE REVIEW

**Comment by jeremyroman** [comment]
The build doesn't pass here because the w3c/resource-timing#343 hasn't been merged (and so there's no definition of "delivery type" to link to).

**Comment by jeremyroman** [comment]
Now that https://github.com/w3c/resource-timing/pull/343 is merged, this can be reviewed now.

File: prefetch.bs [hunk 1]
```diff
@@ -253,6 +253,12 @@ A <dfn export>cross-origin prefetch IP anonymization policy</dfn> has an <dfn ex
 
 <h2 id="html-patches">HTML Patches</h2>
 
+This section contains patches to [[HTML]].
+
+Add an additional item to [=navigation params=] as follows:
+:  <dfn for="navigation params">delivery type</dfn>
+:: a [=string=] (corresponding to {{PerformanceResourceTiming}} [=PerformanceResourceTiming/delivery type=]), default ""
```
[the remark this answers is not in the document]
Comment (domenic) [reply prefetch.bs:340]: This remains unaddressed

File: prefetch.bs [hunk 2]
```diff
@@ -327,6 +333,12 @@ A <dfn export>cross-origin prefetch IP anonymization policy</dfn> has an <dfn ex
...
+
 <div algorithm>
     <div class="note">This is extracted from <a spec=HTML>create navigation params by fetching</a>.</div>
```
Comment (domenic) [inline prefetch.bs:343]: This note now looks like it's applying to the navigation params patch. Maybe add an `<hr>` before the `<div algorithm>`.

**Review by domenic — APPROVED** [review]
Looks good with nits

File: prefetch.bs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeremyroman) [reply prefetch.bs]: Addressed.

File: prefetch.bs (hunk 2 above)
Comment (jeremyroman) [reply prefetch.bs:347]: Done.

**Pull request merged**